### PR TITLE
[13.0][FIX] account_payment_order: avoid cache miss in computed field

### PR DESCRIPTION
**Pull request by astirpe**
We should always ensure to assign something in a compute method

### CODE REVIEW

File: account_payment_order/models/account_move_line.py [hunk 1]
```diff
@@ -42,6 +42,8 @@ def _compute_partner_bank_id(self):
                 )
             ):
                 ml.partner_bank_id = ml.move_id.invoice_partner_bank_id.id
+            else:
+                ml.partner_bank_id = False
```
Comment (pedrobaeza) [inline account_payment_order/models/account_move_line.py]: Being stored=True, it shouldn't be needed. But anyway, it's better to assign the same previous value:
```suggestion
                ml.partner_bank_id = ml.partner_bank_id
```

Comment (pedrobaeza) [reply account_payment_order/models/account_move_line.py]: Maybe the problem is the absence of the default?

**Comment by pedrobaeza** [comment]
Thinking more on this, I believe the problem is the absence of the default, and with my first proposed change is still not going to work. Do you have any test to exercise the problem?

**Comment by astirpe** [comment]
I'm not able to trigger the error with an automated test.
Anyway this is a well-known issue of V13 and is not fixable by adding a default. In Odoo V13 the computed fields were all fixed the same way, see for example [this fix](https://github.com/odoo/odoo/pull/48947/files) and [this other fix](https://github.com/odoo/odoo/pull/38463/files#diff-a6a62c953c1d9ceb1ccf6a3ec670578bR41-R42).

**Comment by pedrobaeza** [comment]
I think the problem is the absence of a default value. Can it be?

**Review by pedrobaeza — APPROVED** [review]
Anyway, as we assign the same value, and this is avoiding the problem, it's OK for me.

**Review by CasVissers-360ERP — APPROVED** [review]
Faced the same issue. Fixed the problem.

**Pull request merged**